### DISCUSSION TITLE
Please add Bluetooth support

### DISCUSSION THREAD

**Discussion by Zero3K** [Genymobile/feedback#23] [Genymotion Desktop] [answered]
It is already possible to use Bluetooth under Android-X86 when it is emulated by VirtualBox (by adding the Bluetooth device to the list of USB devices to be available in the Guest OS). So, it shouldn't be too difficult to add (just have Genymotion Desktop detect the presence of the Bluetooth device, add it to the list, then have the Android OS automatically detect and use it).

**Comment by Zero3K** [comment]
I tried adding the Bluetooth device to the USB section of the VM that was made by Genymotion Desktop. It seems to recognize it (it shows up as Bluetooth under the Connected Devices page of the Settings app). When I try to go to that page, the Settings app crashes.

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @Zero3K ,

Preliminary Bluetooth support is on our roadmap. Though I cannot provide you with an ETA, we may be able to release it in 2023.
